src/Integration.UnitTests/Binding/BindingWorkflowTests.cs

```diff
@@ -173,13 +187,15 @@ public void BindingWorkflow_InstallPackages_Cancellation()
             var progressEvents = new ConfigurableProgressStepExecutionEvents();
             var cts = new CancellationTokenSource();
 
-            ProjectMock project1 = new ProjectMock("project1");
-            ProjectMock project2 = new ProjectMock("project2");
+            ProjectMock project1 = new ProjectMock("project1") { ProjectKind = ProjectSystemHelper.CSharpProjectKind };
+            ProjectMock project2 = new ProjectMock("project2") { ProjectKind = ProjectSystemHelper.CSharpProjectKind };
 
             var nugetPackage = new PackageName("mypackage", new SemanticVersion("1.1.0"));
             var packages = new[] { nugetPackage };
+            var nugetPackages = new Dictionary<Language, IEnumerable<PackageName>>();
```

Comment:
`nugetPackagesByLanguage` would express a lot better what this is. And similarly for the other tests which use the same variable name.